docs/platform_management_plan/project_management.rst

```diff
@@ -181,4 +181,216 @@ meetings, where such topics are discussed and decided, see `Steering committees`
+
+In the main *SCORE* project we use milestones to mark important stages of the project and map sagas or in some cases also other *GitHub Issues* to them.
+
+*Releases* are used for structuring of the development activities. Exact scheme for the releases of the *SCORE* will be provided here later.
+
+You can find "up to date" overview of the release plan and milestones in the following section `Score Releases <https://eclipse-score.github.io/score/score_releases/index.html>`_.
+
+Planning process
+----------------
+Generally, every team is responsible for planning and managing of its backlog.
+For small improvements or clarifications, you can create *GitHub Issue* with a exhaustive
+description and map it to the topic using labels. For small improvements/bugs
+in the software modules you should create *GitHub Issues* directly in the repository
+of the submodule. The project leads and committers of the corresponding software module,
+circle or community will check the issue and in case they will accept it, they will
+take it over to one of their *GitHub Projects*. In case, the topic, that you raise in the issue has a big impact on the platform, you can be asked by the committers to raise a *Contribution Request*.
+
+*Contribution Request* is the central concept to introduce any important changes
+to the project. It is described in more details in *Contribution Request* (TODO: link).
+In general, everyone who wants to provide something new to the project, e.g. a new feature
+or a tool, should provide an exhaustive description, requirements and in some cases
+also initial draft of the architecture as part of the *Contribution Request*.
+The *Contribution Requests* are regularly reviewed in the *Technical lead circle*
+and then get either accepted or declined.
+
+After the *Contribution Request* was accepted, then the *Pull Request* with the
+*Contribution Request* gets merged. The corresponding *GitHub Issue* gets a reference to the
+newly defined saga and gets closed. The saga is at the beginning in the state *"Draft"*. Please be aware, that "status" of the tickets is modelled in *GitHub Project* as *GitHub Issues* do not provide the possibility to define additional states.
+
+The *Technical lead circle* is responsible for maintenance of the backlog with sagas,
+their prioritization and creation of the roadmap. Together with software module
+project leads and community leads in the "Committer circle" they go through the backlog, decide when and which saga should be implemented in which order and update the roadmap accordingly.
+
+As soon as the saga was planned for implementation, its state is changed to *"Open"*.
+As next step, a *GitHub Issue* of type *epic* is created as sub-issue of the saga
+and gets assigned to one of the *Communities* for *Feature Architecture Definition*.
```

Comment:
now this contradicts the part after the image. Better to say here "for refinement" instead of "Feature Architecture" (which is also not true for processes and tools).